modules/ROOT/pages/box/box-connector-examples.adoc

```diff
@@ -250,7 +253,295 @@ output application/json
 </mule>
 ----
 
+== Example: Upload a File
+
+This example demonstrates how to upload a file to Box.
+
+. Create a new Mule Project in Anypoint Studio.
```

Comment:
It might be labeled "Mule Project" in the UI, but what we are creating is "a Mule project." Does that distinction make sense?